Logger/BaseLoggerMixins.cs

```diff
@@ -1,7 +1,37 @@
-﻿namespace Logger
+﻿using System;
+
+namespace Logger
 {
     public static class BaseLoggerMixins
     {
-        
+        public static void Error(this BaseLogger? log, string? message, params object[]? arguments)
```

Comment:
```suggestion
        public static void Error(this BaseLogger? log, string message, params object[]? arguments)
```
This could be non-nullable.